src/domain/events/eventsTable/eventsTableRow/EventsTableRow.tsx

```diff
@@ -134,7 +134,11 @@ const EventTableRow: React.FC<Props> = ({
           </div>
         </td>
         <td className={styles.publisherColumn}>
-          <OrganizationName id={publisher} />
+          {publisher !== null ? (
```

Comment:
For external users publisher is automatically set to "others", so should check if publisher does not equal "others" 🤔